src/System.Composition.Convention/src/System/Composition/Convention/PartConventionBuilder.cs

```diff
@@ -541,7 +541,7 @@ internal bool BuildConstructorAttributes(Type type, ref List<Tuple<object, List<
             foreach (var ci in constructors)
             {
                 // We have a constructor configuration we must log a warning then not bother with ConstructorAttributes
-                IEnumerable<Attribute> attributes = ci.GetCustomAttributes(typeof(ImportingConstructorAttribute), false);
+                IEnumerable<Attribute> attributes = Attribute.GetCustomAttributes(ci, typeof(ImportingConstructorAttribute), false);
```

Comment:
Since this is now compiling against the latest contracts, these methods were binding against different overloads. This should be the same exact thing as before, but using an explicit call rather than an extension method call.